### PR TITLE
check if one entry or not

### PR DESCRIPTION
**Pull request by nicosomb**

### CODE REVIEW

**Comment by modos189** [comment]
Filters by create date are delete =(

**Comment by nicosomb** [comment]
oh yes, because we have to merge #1343 before.

File: src/Wallabag/CoreBundle/Resources/views/themes/material/Entry/entries.html.twig [hunk 1]
```diff
@@ -17,7 +17,7 @@
     {% block pager %}
     {% if entries is not empty %}
         <div class="results clearfix">
-            <div class="nb-results left">{{ entries.count }} {% trans %}entries{% endtrans %}</div>
+            <div class="nb-results left">{{ entries.count }} {% trans %}entrie{% endtrans %}{% if entries.nbPages > 1 %}s{% endif %}</div>
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/views/themes/material/Entry/entries.html.twig]: Use [`transchoice `](http://symfony.com/doc/current/book/translation.html#twig-templates) instead

Comment (nicosomb) [inline src/Wallabag/CoreBundle/Resources/views/themes/material/Entry/entries.html.twig]: wow <3

**Pull request merged**